File: CoderCave/client/src/components/tag/Tag.js
import { Link } from "react-router-dom";
import { Card, CardBody, CardText, CardTitle } from "reactstrap";

const Tag = ({ tag }) => {
  return (
    <Card>
      <CardBody>
        <CardTitle><Link to={`/tag/details/${tag.id}`}>{tag.name}</Link></CardTitle>
        <CardText>{tag.description}... </CardText>
        <CardText><Link to={`/tag/details/${tag.id}`}>Read More</Link></CardText>
        <CardText>{tag.inquireCount} Questions</CardText>
      </CardBody>
    </Card>
  );
};

export default Tag;

File: CoderCave/Models/Tag.cs
using System.ComponentModel.DataAnnotations;

namespace CoderCave.Models
{
    public class Tag
    {
        [Required]
        public int Id { get; set; }
        
        [Required]
        public string Name { get; set; }
        
        [Required]
        public string Description { get; set; }
        public int InquireCount { get; set; }
    }
}
